src/main/java/org/cactoos/collection/TailOf.java

```diff
@@ -52,16 +51,7 @@ public TailOf(final int num, final T... src) {
      * @param src Source iterable
      */
     public TailOf(final int num, final Iterable<T> src) {
-        this(num, new CollectionOf<T>(src));
-    }
-
-    /**
-     * Ctor.
-     * @param num Number of tail elements
-     * @param src Source collection
-     */
-    public TailOf(final int num, final Collection<T> src) {
-        super(() -> new CollectionOf<T>(
+        super(new Sticky<T>(
```

Comment:
@victornoel please, use the same indentation here which you used bellow (IMHO it's better).